main.go

```diff
@@ -347,6 +349,20 @@ func newController(klient kubernetes.Interface, logger log.Logger, o *options) *
 		},
 			[]string{"type"},
 		),
+		hashringNodes: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "thanos_receive_controller_hashrings_nodes",
```

Comment:
nit: can we remove the `s` at the end of `hashrings` to name the metric `thanos_receive_controller_hashring_nodes`